Cargo.toml

```diff
@@ -13,20 +13,23 @@ publish = false
 
 [dependencies]
 anyhow = "1"
-ceramic-event = { git = "https://github.com/3box/rust-ceramic", branch = "main" }
-json-patch = { version = "1.0.0", features = ["diff"] }
+#ceramic-event = { git = "https://github.com/ceramicnetwork/rust-ceramic", branch = "main" }
```

Comment:
Maybe delete the unused dependencies? Or I guess we could merge the wasi changes and go back to main.

Reply:
Kind of would like to just merge the wasi changes. Otherwise I can delete.

Reply:
I don't have objections, it looks like that one needs a rebase and format but should be fine. 

Was curious if we have any policy around forks (3box, ceramicnetwork, personal account).. obviously it'd be nicer to not have them, and we don't want to give the impression we're maintaining something when we just want a fix